VContainer/Assets/VContainer/Runtime/Unity/LifetimeScope.cs

```diff
@@ -221,8 +230,15 @@ public LifetimeScope CreateChildFromPrefab(LifetimeScope prefab, IInstaller inst
             return child;
         }
 
+        public LifetimeScope CreateChildFromPrefab(LifetimeScope prefab, IInstaller installer = null)
```

Comment:
I didn't see the need for both non-generic and generic methods.

If there is a generic method, it should be callable, like CreateChildFromPrefab(lifetimeScope).

Can we unify only the generic method, like CreateChild()?

Reply:
I've removed redundant CreateChildFromPrefab() overloads. 

Can't do the same for CreateChild<TScope>() without breaking backwards compatibility. 
It doesn't have a parameter to infer TScope type automatically.